Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM node:lts-alpine as builder
+
+COPY . /src
+WORKDIR /src
+
```

Comment:
To make it compile on my laptop i have to add this:
`
ENV NODE_OPTIONS="--max-old-space-size=8192"
`
Although it has far enough space it fails without that. Might be worth adding it to avoid headache to others?

Reply:
Interesting… …never saw that option anywhere… Had to [read about it](https://nodejs.org/api/cli.html#--max-old-space-sizesize-in-megabytes)…

How much memory does your machine have? And can you compile the application without that option when not using Docker? 🤔

Reply:
I actually had to do this to compile it locally as well yes.
I have 16GB of RAM and ~40GB of disk space.
But maybe i set an env var in the past i forgot about..

Reply:
Hmm interesting… On my system the compile uses quite a lot of memory but not remotely near 8Gi:

```
# /usr/bin/time npm run build

> twitchat@11.2.1 build
> npm run i18n && npm run front/build && npm run server/build
...
> twitchat@11.2.1 server/build
> tsc -p ./tsconfig_server.json

57.67user 7.69system 0:32.94elapsed 198%CPU (0avgtext+0avgdata 3126192maxresident)k
64inputs+122032outputs (0major+1563386minor)pagefaults 0swaps
```

I mean, 3.13Gi memory usage is definitely a lot but why is it running against a limit on your system… 🤔